wagtail/wagtailforms/templates/wagtailforms/list_submissions.html

```diff
@@ -10,16 +10,22 @@
             {% for heading in data_headings %}
                 <th>{{ heading }}</th>
             {% endfor %}
+            <th>{% trans "Delete Submission" %}</th>
```

Comment:
Thank you for your comment, @MechanisM. 'Actions' seems to be more appropriate in this case. and the shorter column name will be better when the table layout shrinks with many columns.